.claude/skills/membrane-config/scripts/describe_element.py

```
@@ -0,0 +1,127 @@
+            d = parent
+    return None
+
+
+def load_schema():
+    path = find_local_schema()
+    if not path:
+        os.makedirs(CACHE_DIR, exist_ok=True)
+        path = os.path.join(CACHE_DIR, "v7.2.3.json")
+        if not os.path.isfile(path):
+            print(f"Downloading schema from {PUBLISHED_SCHEMA_URL} ...", file=sys.stderr)
+            urllib.request.urlretrieve(PUBLISHED_SCHEMA_URL, path)
```

Ruff (0.15.18)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)